_install-and-configure/upgrade-opensearch/rolling-upgrade.md

```diff
@@ -0,0 +1,196 @@
+1. Confirm that the associated node has been dismissed from the cluster.
+   ```bash
+   curl -s "http://localhost:9202/_cat/nodes?v&h=name,version,node.role,master" | column -t
+   ```
+   Sample output:
+   ```bash
+   name        version  node.role  master
+   os-node-02  7.10.2   dimr       *
+   os-node-04  7.10.2   dimr       -
+   os-node-03  7.10.2   dimr       -
+   ```
+   `os-node-01` is no longer listed because the container has been stopped and deleted.
+1. Deploy a new container running the desired version of OpenSearch, mapped to the same volume as the container you deleted.
+1. Query the `_cat/nodes` endpoint after OpenSearch is running on the new node to confirm that it has joined the cluster.
+   ```bash
+   curl -s "http://localhost:9201/_cat/nodes?v&h=name,version,node.role,master" | column -t
+   ```
+   Sample output:
+   ```bash
+   name        version  node.role  master
+   os-node-02  7.10.2   dimr       *
+   os-node-04  7.10.2   dimr       -
+   os-node-01  7.10.2   dimr       -
+   os-node-03  7.10.2   dimr       -
+   ```
+   In the sample output, the new OpenSearch node reports a running version of `7.10.2` to the cluster. This is the result of `compatibility.override_main_response_version`, which is used when connecting to a cluster with legacy clients that check for a version. You can manually confirm the version of the node by calling the `/_nodes` API endpoint, as in the following command. Replace `<nodeName>` with the name of your node. See [Nodes API]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/index/) to learn more.
+   ```
+   curl -s -X GET 'localhost:9201/_nodes/<nodeName>?pretty=true' | jq -r '.nodes | .[] | "\(.name) v\(.version)"'
+   ```
+   Sample output:
+   ```
+   $ curl -s -X GET 'localhost:9201/_nodes/os-node-01?pretty=true' | jq -r '.nodes | .[] | "\(.name) v\(.version)"'
+   os-node-01 v1.3.7
+   ```
+1. Repeat steps 5 through 9 for each node in your cluster. Remember to upgrade an eligible cluster manager node last. After replacing the last node, query the `_cat/nodes` endpoint to confirm that all nodes have joined the cluster. The cluster is now bootstrapped to the new version of OpenSearch.
+   ```bash
+   curl -s "http://localhost:9201/_cat/nodes?v&h=name,version,node.role,master" | column -t
+   ```
+   Sample output:
+   ```bash
+   name        version  node.role  master
+   os-node-04  1.3.7    dimr       -
+   os-node-02  1.3.7    dimr       *
+   os-node-01  1.3.7    dimr       -
+   os-node-03  1.3.7    dimr       -
+   ```
+1. Reenable shard replication.
+   ```bash
+   curl -X PUT "http://localhost:9201/_cluster/settings?pretty" -H 'Content-type: application/json' -d'{"persistent":{"cluster.routing.allocation.enable":"all"}}'
+   ```
+   Sample output:
```

Comment:
Rewrite as a complete sentence and use "example" instead of "sample".

Reply:
Pinged for feedback. Idea: `The response should look like the following example:`

Reply:
Settled on a great suggestion from Nathan: `The response should look similar to the following example:`